<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>731</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_main">
         <property name="text">
          <string>首页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_co">
         <property name="text">
          <string>收藏</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_mine">
         <property name="text">
          <string>我的</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="mine">
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>621</width>
           <height>371</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,9">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0">
            <item alignment="Qt::AlignmentFlag::AlignLeft">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>肖</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignmentFlag::AlignHCenter">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>育</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignmentFlag::AlignRight">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>璇</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGraphicsView" name="graphicsView"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="main">
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>3</y>
           <width>531</width>
           <height>271</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
            <property name="leftMargin">
             <number>4</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="rightMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
              <property name="sizeConstraint">
               <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
              </property>
              <property name="leftMargin">
               <number>4</number>
              </property>
              <property name="topMargin">
               <number>4</number>
              </property>
              <property name="rightMargin">
               <number>4</number>
              </property>
              <property name="bottomMargin">
               <number>4</number>
              </property>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="sizeIncrement">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>输入关键字：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPlainTextEdit" name="plainTextEdit">
                <property name="maximumSize">
                 <size>
                  <width>400</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="baseSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="mouseTracking">
                 <bool>false</bool>
                </property>
                <property name="plainText">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_search">
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="text">
                 <string>搜索</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QListWidget" name="listWidget">
              <property name="maximumSize">
               <size>
                <width>580</width>
                <height>400</height>
               </size>
              </property>
              <property name="contextMenuPolicy">
               <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
              </property>
              <property name="resizeMode">
               <enum>QListView::ResizeMode::Fixed</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="collector">
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>601</width>
           <height>534</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetFixedSize</enum>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
            <property name="leftMargin">
             <number>4</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="rightMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>我的收藏</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listWidget_2">
              <property name="contextMenuPolicy">
               <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>752</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <action name="action">
   <property name="text">
    <string>  </string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
